src/discord/discord_state.rs

```diff
@@ -171,8 +171,9 @@ impl DiscordState {
         self.has_admin_role(user_id, self.gid.clone()).await
     }
 
-    pub async fn get_user(&self, user_id: Id<UserMarker>) -> Result<Option<User>, String> {
-        Ok(self.cache.user(user_id).map(|u| u.value().clone()))
+    /// convenience method for getting a user's info from the twilight cache
+    pub async fn get_user(&self, user_id: Id<UserMarker>) -> Option<User> {
```

Comment:
i think i was imagining that this method might one day fallback to actual API requests but I no longer really think that